.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -1,16 +0,0 @@
-Before you submit a pull request, please fill in the following:
```

Comment:
Why are we removing the PR template?

Reply:
It was also annoying because it was failing linting 😛

Reply:
I takeback what I said previously, they didn't update new templates